File: 3065.minimum-operations-to-exceed-threshold-value-i.py
#
# @lc app=leetcode id=3065 lang=python
#
# [3065] Minimum Operations to Exceed Threshold Value I
#

# @lc code=start
class Solution(object):
    def minOperations(self, nums, k):
        """
        :type nums: List[int]
        :type k: int
        :rtype: int
        """
        c = 0
        for i in nums:
            if i < k:
                c += 1
        return c
        
# @lc code=end

File: 3065.minimum-operations-to-exceed-threshold-value-i.java
/*
 * @lc app=leetcode id=3065 lang=java
 *
 * [3065] Minimum Operations to Exceed Threshold Value I
 */

// @lc code=start
class Solution {
    public int minOperations(int[] nums, int k) {
       int c = 0;
    for (int i : nums) {
    if (i < k) {
        c++;
      }
    }
    return c;
     
    }
}
// @lc code=end

